src/app/zap-templates/zcl/data-model/chip/electrical-energy-measurement-cluster.xml

```diff
@@ -0,0 +1,108 @@
+    <item name="Voltage" value="1" />
+    <item name="ActiveCurrent" value="2" />
+    <item name="ReactiveCurrent" value="3" />
+    <item name="ApparentCurrent" value="4" />
+    <item name="ActivePower" value="5" />
+    <item name="ReactivePower" value="6" />
+    <item name="ApparentPower" value="7" />
+    <item name="RMSVoltage" value="8" />
+    <item name="RMSCurrent" value="9" />
+    <item name="RMSPower" value="10" />
+    <item name="Frequency" value="11" />
+    <item name="PowerFactor" value="12" />
+    <item name="NeutralCurrent" value="13" />
+    <item name="ElectricalEnergy" value="14" />
+  </enum>
+
+  <struct name="EnergyMeasurementStruct">
+    <cluster code="0x0091"/>
+    <!-- TODO: Energy should be energy_mwh once supported -->
+    <item fieldId="0" name="Energy" type="int64s" />
+    <item fieldId="1" name="StartTimestamp" type="epoch_s" optional="true"/>
+    <item fieldId="2" name="EndTimestamp" type="epoch_s" optional="true"/>
+    <item fieldId="3" name="StartSystime" type="systime_ms" optional="true"/>
+    <item fieldId="4" name="EndSystime" type="systime_ms" optional="true"/>
+  </struct>
+
+  <struct name="MeasurementAccuracyRangeStruct">
+    <cluster code="0x0091"/>
+    <item fieldId="0" name="RangeMin" type="int64s" />
+    <item fieldId="1" name="RangeMax" type="int64s" />
+    <item fieldId="2" name="PercentMax" type="percent100ths" optional="true" />
+    <item fieldId="3" name="PercentMin" type="percent100ths"  optional="true"/>
+    <item fieldId="4" name="PercentTypical" type="percent100ths"  optional="true"/>
+    <item fieldId="5" name="FixedMax" type="int64u"  optional="true"/>
+    <item fieldId="6" name="FixedMin" type="int64u"  optional="true"/>
+    <item fieldId="7" name="FixedTypical" type="int64u"  optional="true"/>
+  </struct>
+
+  <struct name="MeasurementAccuracyStruct">
+    <cluster code="0x0091"/>
+    <item fieldId="0" name="MeasurementType" type="MeasurementTypeEnum" />
+    <item fieldId="1" name="Measured" type="boolean" />
+    <item fieldId="2" name="MinMeasuredValue" type="int64s" />
+    <item fieldId="3" name="MaxMeasuredValue" type="int64s" />
+    <item fieldId="4" name="AccuracyRanges" type="MeasurementAccuracyRangeStruct" array="true" />
+  </struct>
+
+  <cluster apiMaturity="provisional">
+    <name>Electrical Energy Measurement</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>This cluster provides a mechanism for querying data about the electrical energy imported or provided by the server.</description>
+    <code>0x0091</code>
+    <define>ELECTRICAL_ENERGY_MEASUREMENT_CLUSTER</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+    <globalAttribute side="either" code="0xFFFD" value="1"/> <!-- Revision   -->
+
+    <attribute side="server" code="0x0000" define="MEASURED" type="boolean" writable="false" default="false" optional="false">Measured</attribute>
```

Comment:
I think MEASURED has been deleted in latest spec? so all of the IDs need shuffling up.